eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -43,10 +42,22 @@ jobs:
       - ${{ if not(contains(variables['Build.DefinitionName'], '-pr - ')) }}:
         - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
           parameters:
-              Paths:
-                - "/*"
-                - "!SessionRecords"
-                - "/sdk/${{ parameters.ServiceToTest }}/**/SessionRecords"
+            Paths:
+              - "/*"
+              - "!SessionRecords"
+              - "/sdk/*/**/SessionRecords"
```

Comment:
We definitely don't want to default this to be * all the time now as that will pull in all test recordings all the time. We will want to use ServiceDirectory whenever ProjectListOverrideFileProperty is set to empty.

Reply:
Done.